packs/equipment/reflecting-shield.json

```
@@ -35,7 +35,18 @@
             "title": "Pathfinder GM Core"
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "self:shield:raised",
+                    "spell"
```

```suggestion
                    "item:type:spell"
```
We should avoid using top level roll options when possible.

We also need to make this a toggle, since it's a reaction so it will not apply to all spells when the shield is raised.

---

The rule element is for the effect from the spellguard shield, which is active against all spells that target you, not for the reaction.
If any part of it needs changing, it should be changed on spellguard shield as well.